ui/src/components/TaskRunsCard/index.jsx

```diff
@@ -252,13 +253,7 @@ export default class TaskRunsCard extends Component {
 
     return {
       ...artifacts,
-      edges: [...artifacts.edges].sort((a, b) => {
-        if (a.node.isPublic === b.node.isPublic) {
```

Comment:
btw, search code base didn't revel any place where this value was set. There were only some checks on the queue api side, but it was never exposed, so I wonder if it ever worked at all

Reply:
I recall this property (`isPublic`) was calculated in a weird way in a weird place, and I think I removed that.  Maybe I missed this spot.

Reply:
Thanks! I decided that calculating it in the UI is enough for now, so didn't change API returned object.